spring/boot-autoconfigure/build.gradle

```diff
@@ -15,5 +15,7 @@ dependencies {
     testImplementation project(':grpc')
     testImplementation project(':thrift')
     testImplementation 'org.springframework.boot:spring-boot-starter-actuator'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation ('org.springframework.boot:spring-boot-starter-test') {
+        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+    }
```

Comment:
I changed all test cases in `spring-boot-autoconfigure` to JUnit5. The test ran twice, showing unintended failures in certain situations.

Reply:
> JUnit 5

IIRC, Spring Boot 1 does not support JUnit 5 officially, so we should stay with JUnit 4.

> I think we should never specify a certain port when testing. Can you just specify port `0` to use an ephemeral port?

Reply:
> IIRC, Spring Boot 1 does not support JUnit 5 officially, so we should stay with JUnit 4.

Oops ... I couldn't confirm this.

> I think we should never specify a certain port when testing. Can you just specify port 0 to use an ephemeral port?

I'll check. Thank you. 😄